third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -3171,6 +3211,37 @@ resource "google_container_cluster" "with_pod_security_policy" {
 	}
 }`, clusterName, enabled)
 }
+
+func testAccContainerCluster_withResourceUsageExportConfig(clusterName, datasetId string, resourceUsage, egressMetering bool) string {
```

Comment:
egressMetering is a parameter but the test always sets it to false. That intentional?

My ideal test would set up the cluster with ResourceUsageExportConfig (since users are probably more likely to set it on create than on update), and then modify it in some way, rather than it going from unset to set, but up to you- it's not a strong preference.

Reply:
> egressMetering is a parameter but the test always sets it to false. That intentional?

I was originally thinking of testing multiple combinations of the resourceUsage attribute but end up only testing the the creation and updates of clusters with and without resource usage enabled. I updated the code so that that test always set this to `true` now. lmk what you think about that

> My ideal test would set up the cluster with ResourceUsageExportConfig (since users are probably more likely to set it on create than on update), and then modify it in some way, rather than it going from unset to set, but up to you- it's not a strong preference.

That's pretty much what the current test is doing. it creates a cluster with resource usage enabled and then fires and update to disable it